src/SourceBuild/tarball/content/repos/known-good.proj

```diff
@@ -35,17 +35,17 @@
         <RepositoryReference Include="diagnostics" />
         <RepositoryReference Include="llvm-project" />
         <RepositoryReference Include="razor-compiler" />
+        <RepositoryReference Include="xliff-tasks" />
```

Comment:
Consider making a separate PR to check in these changes since ideally this PR would be abandoned because all the patches would get backported.